cpg_infra/driver.py

```diff
@@ -1707,6 +1807,7 @@ def setup_legacy_container_registries(self):
         }
 
         for kind, account in kinds.items():
+            # TODO, this can be solved on high level by adding common-images-reader to analysis-runner container registry
```

Comment:
Did you mean to resolve this?

Reply:
I wasn't going to, but have since resolved.